realm/realm-library/src/androidTest/java/io/realm/RealmListTests.java

```diff
@@ -763,8 +779,67 @@ public void realmMethods_onDeletedLinkView() {
     }
 
     @Test
-    public void removeAllFromRealm() {
+    public void deleteFromRealm() {
```

Comment:
we just need add assertion to `ManagedOrderedRealmCollectionTests`. I am fixing it now